src/plutus/cek.md

```diff
@@ -0,0 +1,136 @@
+# The CEK Machine
+
+This page presents the operational semantics of the Plutus language using an abstract machine - a [CEK machine](https://en.wikipedia.org/wiki/CEK_Machine).
+Although more complex than traditional reduction semantics, the CEK machine offers an efficient evaluation strategy that enables precise modeling of resource usage and cost.
+The production Haskell evaluator is based on the CEK machine, and it also provides a practical foundation for alternative implementations.
+
+The following listing defines some key concepts of the CEK machine.
+
+```text
+Σ ∈ State ::= 𝑠; 𝜌 ⊳ 𝑀  // Computing M under environment 𝜌 with stack 𝑠
```

Comment:
It might look neater if the `//`s were aligned vertically.  Also, maybe we could just have text without the comment symbols.

Reply:
Not possible to align precisely since unicode characters are not monospace. Removing the `//` sounds good - will do